README.md

```diff
@@ -34,60 +36,43 @@
 
 ## Usage
 
-### [BetterDiscord](https://betterdiscord.app)
-
-1. Download your preferred flavour:
-
-- 🌻 [Latte](./themes/latte.theme.css?raw=1)
-- 🪴 [Frappe](./themes/frappe.theme.css?raw=1)
-- 🌺 [Macchiato](./themes/macchiato.theme.css?raw=1)
-- 🌿 [Mocha](./themes/mocha.theme.css?raw=1)
-
-2. Copy the downloaded file to your BetterDiscord themes folder.
-3. Enable the theme in BetterDiscord settings.
-
-### Clients/Mods with custom CSS support
+### Custom CSS ([Vencord](https://vencord.dev/), [BetterDiscord](https://betterdiscord.app/))
 
-1. Simply add your preferred flavour into your discord clients CustomCSS file/editor.
+Add `@import url("https://catppuccin.github.io/discord/dist/catppuccin-<flavor>-<accent>.theme.css");` to your Discord client's custom CSS file/editor, where `<flavor` and `<accent>` are your preferred flavor and accent, respectively. To simply go with the default accent color, omit the `-<accent>` part. For example:
```

Comment:
"Omit accent to default to blue."

Reply:
Would you like to make this a suggestion with a diff so we don't have further back and forth over this language?